frontend/public/components/storage/attach-storage.tsx

```diff
@@ -160,186 +170,160 @@ class AttachStorageForm extends React.Component<
+          </label>
+          <div>
+            <input
+              className="pf-c-form-control"
+              type="text"
+              onChange={handleMountPathChange}
+              aria-describedby="mount-path-help"
+              name="mountPath"
+              id="mount-path"
+              value={mountPath}
+              required
+            />
+            <p className="help-block" id="mount-path-help">
+              Mount path for the volume inside the container.
+            </p>
           </div>
-          <Checkbox
-            label="Mount as read-only"
-            onChange={this.onMountAsReadOnlyChanged}
-            checked={this.state.mountAsReadOnly}
-            name="mountAsReadOnly"
-          />
-          <div className="form-group">
-            <label className="control-label" htmlFor="subpath">
-              Subpath
-            </label>
-            <div>
-              <input
-                className="pf-c-form-control"
-                type="text"
-                onChange={this.handleChange}
-                aria-describedby="subpath-help"
-                id="subpath"
-                name="subPath"
-                value={subPath}
-              />
-              <p className="help-block" id="subpath-help">
-                Optional path within the volume from which it will be mounted
-                into the container. Defaults to the root of volume.
-              </p>
-            </div>
+        </div>
+        <Checkbox
+          label="Mount as read-only"
+          onChange={onMountAsReadOnlyChanged}
+          checked={mountAsReadOnly}
+          name="mountAsReadOnly"
+        />
+        <div className="form-group">
+          <label className="control-label" htmlFor="subpath">
+            Subpath
+          </label>
+          <div>
+            <input
+              className="pf-c-form-control"
+              type="text"
+              onChange={handleSubPathChange}
+              aria-describedby="subpath-help"
+              id="subpath"
+              name="subPath"
+              value={subPath}
+            />
+            <p className="help-block" id="subpath-help">
+              Optional path within the volume from which it will be mounted
+              into the container. Defaults to the root of volume.
+            </p>
           </div>
-          <ButtonBar errorMessage={error} inProgress={inProgress}>
-            <button type="submit" className="btn btn-primary" id="save-changes">
-              Save
-            </button>
-            <button
-              type="button"
-              className="btn btn-default"
-              onClick={history.goBack}
-            >
-              Cancel
-            </button>
-          </ButtonBar>
-        </form>
-      </div>
-    );
-  }
-}
+        </div>
+        {!useContainerSelector && <p>The volume will be mounted into all containers. You can <a onClick={handleSelectContainers}>select specific containers</a> instead.</p>}
```

Comment:
`button` is better for accessibility. `a` elements with no `href` can't be focused with the keyboard or a screen reader.

Avoid long lines

```suggestion
        {!useContainerSelector && <p>The volume will be mounted into all containers. You can <button type="button" className="btn btn-link .btn-link--no-btn-default-values" onClick={handleSelectContainers}>select specific containers</button> instead.</p>}
```